src/components/theme/default/index.js

```diff
@@ -1,15 +1,56 @@
 import { createMuiTheme } from "@material-ui/core/styles";
 import { palette } from "@cyverse-de/ui-lib";
 
+const cyverseColors = {
```

Comment:
Yep, I'm pretty sure these are already defined in `ui-lib` (though I think that cyverse-de/ui-lib#28 PR was just merged today).

Reply:
Yeah, I haven't rebased and corrected this yet.